src/uncontrast/pages/ParametersDistributionPie/ParametersDistributionPie.jsx

```diff
@@ -55,6 +55,14 @@ export default function ParametersDistributionPie() {
     "#701960",
     "#088515",
     "#496B9B",
+    "#6789B9",
```

Comment:
please this is a bug just waiting to happen, eventually need would add more categories and ..

Reply:
i took care of it so on the use of that list it will cycle when gets to array end. on the next commit